fbw-a32nx/src/systems/instruments/src/PFD/instrument.tsx

```diff
@@ -71,6 +81,10 @@ class A32NX_PFD extends BaseInstrument {
     this.backplane.addPublisher('Dmc', this.dmcPublisher);
     this.backplane.addPublisher('RopRowOans', this.ropRowOansPublisher);
     this.backplane.addPublisher('Fwc', this.fwcPublisher);
+    this.backplane.addPublisher('FcuBus', this.fcuBusPublisher);
+    this.backplane.addPublisher('ElectricalSystem', this.elecSystemPublisher);
+    this.backplane.addPublisher('FwcBus', this.fwcBusPublisher);
```

Comment:
Can you check why we have a `fwcPublisher` and a `fwcBusPublisher`?

Reply:
Merge shenanigans. I will prefer to keep the new one as it has more complete typings/style with an extra topic and more docs.